src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/DefendantResponseReceiptContentProvider.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.NotificationsProperties;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.ClaimContentProvider;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.ResponseData;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDate;
+
+@Component
+public class DefendantResponseReceiptContentProvider {
+
+    private final PartyDetailsContentProvider partyDetailsContentProvider;
+    private final ClaimContentProvider claimContentProvider;
+    private final NotificationsProperties notificationsProperties;
+
+    public DefendantResponseReceiptContentProvider(
+        final PartyDetailsContentProvider partyDetailsContentProvider,
+        final ClaimContentProvider claimContentProvider,
+        final NotificationsProperties notificationsProperties
+        ) {
+        this.partyDetailsContentProvider = partyDetailsContentProvider;
+        this.claimContentProvider = claimContentProvider;
+        this.notificationsProperties = notificationsProperties;
+    }
+
+
+    public Map<String, Object> createContent(final Claim claim) {
+        requireNonNull(claim);
+        ResponseData defendantResponse = claim.getResponse().orElseThrow(IllegalStateException::new);
+
+        Map<String, Object> content = new HashMap<>();
+        content.put("claim", claimContentProvider.createContent(claim));
+        content.put("defenceSubmittedOn", formatDate(claim.getRespondedAt()));
+        content.put("defendant", partyDetailsContentProvider.createContent(
+            claim.getClaimData().getDefendant(),
+            defendantResponse.getDefendant(),
+            claim.getDefendantEmail()
+        ));
+        content.put("claimant", partyDetailsContentProvider.createContent(
+            claim.getClaimData().getDefendant(),
+            claim.getClaimData().getClaimant(),
+            claim.getSubmitterEmail()
+        ));
+        Optional<StatementOfTruth> optionalStatementOfTruth = defendantResponse.getStatementOfTruth();
+        content.put("signerName", optionalStatementOfTruth.map((StatementOfTruth::getSignerName)).orElse(null));
+        content.put("signerRole", optionalStatementOfTruth.map((StatementOfTruth::getSignerRole)).orElse(null));content.put("signerRole", optionalStatementOfTruth.map((StatementOfTruth::getSignerRole)).orElse(null));
```

Comment:
accidentally put up in same line looks like

Reply:
Thank you for spotting that. Its been corrected now.